python_modules/libraries/dagster-sdf/dagster_sdf/resource.py

```diff
@@ -456,6 +467,8 @@ def cli(
             SdfCliInvocation: A invocation instance that can be used to retrieve the output of the
                 sdf CLI command.
         """
+        dagster_sdf_translator = validate_opt_translator(dagster_sdf_translator)
+        dagster_sdf_translator = dagster_sdf_translator or DagsterSdfTranslator()
```

Comment:
I believe this line can be removed as the default `DagsterSdfTranslator` of `opt_inst_param` is returned.